### PR TITLE
Add user documentation for path level configuration

### PR DESCRIPTION
**Pull request by cheng-chang**

### CODE REVIEW

**Comment by cheng-chang** [comment]
jenkins, test this please

File: docs/en/basic/Configuration-Settings.md [hunk 1]
```diff
@@ -139,15 +139,34 @@ master's `alluxio-site.properties` file.
...
+Clients can ignore or overwrite the cluster-wide default values by following the approaches described in
+[Configure Alluxio for Applications](#configure-applications) to overwrite the same properties.
 
 > Note that, before version 1.8, `${ALLUXIO_HOME}/conf/alluxio-site.properties` file is only loaded
 by Alluxio server processes and will be ignored by applications interacting with Alluxio service
 through Alluxio client, unless `${ALLUXIO_HOME}/conf` is on applications' classpath.
 
+### Path Defaults
+
+Since version 2.0, Alluxio administrator can set default client side configurations for Alluxio paths.
```
Comment (calvinjia) [inline docs/en/basic/Configuration-Settings.md]: `administrators`

File: docs/en/basic/Configuration-Settings.md [hunk 2]
```diff
@@ -139,15 +139,34 @@ master's `alluxio-site.properties` file.
...
+FileSystem client operations have options, by default, the options are derived from client side configuration
```
Comment (calvinjia) [inline docs/en/basic/Configuration-Settings.md]: remove `by default`? 
`FileSystem client operations have options that are derived from client side configuration`

File: docs/en/basic/Configuration-Settings.md [hunk 3]
```diff
@@ -139,15 +139,34 @@ master's `alluxio-site.properties` file.
...
+properties. Only these configuration properties can be set as as path defaults.
+
+For example, `createFile` has an option to specify write type, by default, the write type is
```
Comment (calvinjia) [inline docs/en/basic/Configuration-Settings.md]: `type. By default,`

**Comment by cheng-chang** [comment]
@calvinjia PTAL

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**